modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -0,0 +1,137 @@
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* Install the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Create the workflow configuration `YAML` file:
++
+For example, create a sample `workflow-dev.yaml` file containing the following sample code:
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: greeting
+  annotations:
+    sonataflow.org/description: Greeting example on k8s!
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: dev 
+spec:
+  flow: 
+    start: ChooseOnLanguage
+    functions:
+      - name: greetFunction
+        type: custom
+        operation: sysout
+    states:
+      - name: ChooseOnLanguage
+        type: switch
+        dataConditions:
+          - condition: "${ .language == \"English\" }"
+            transition: GreetInEnglish
+          - condition: "${ .language == \"Spanish\" }"
+            transition: GreetInSpanish
+        defaultCondition: GreetInEnglish
+      - name: GreetInEnglish
+        type: inject
+        data:
+          greeting: "Hello from JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetInSpanish
+        type: inject
+        data:
+          greeting: "Saludos desde JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetPerson
+        type: operation
+        actions:
+          - name: greetAction
+            functionRef:
+              refName: greetFunction
+              arguments:
+                message:  ".greeting + .name"
+        end: true
+----
+
+. Navigate to the directory where the `YAML` file is contained, and deploy the application by applying the `YAML` file:
++
+[source,terminal]
+----
+$ oc apply -f <filename> -n <your_namespace>
+----
++
+.Example command
+[source,terminal]
+----
+$ oc apply -f workflow-dev.yaml
+----
+
+. Verify the deployment and check the status of the the deployed workflow:
++
+[source,terminal]
+----
+$ oc get workflow -n <your_namespace> -w
+----
++
+.Example command
+[source,terminal]
+----
+$ oc get sonataflows
+----
++
+Ensure that the workflow `greeting` is listed and the status is `Running` or `Completed`.
+
+. Edit or modify the workflow directly in the cluster: 
++
+[source,terminal]
+----
+$ oc edit sonataflow <workflow_name> -n <your_namespace>
+----
++
+.Example command
+[source,terminal]
+----
+$ oc edit sonataflow greeting -n <your_namespace>
+----
+
+. After editing, save the changes. The Openshift Serverless Logic Operator will detect the changes and update the workflow accordingly.
```

Comment:
I would just use the present tense here - "detects" and "updates" without "will"